utils/parameters.js

```diff
@@ -0,0 +1,7 @@
+const mongoose = require("mongoose");
+
+function loan_params () {
+	var maxloans = 1000;
+};
+
+module.exports = loan_params() ;
```

Comment:
This is a mess. First of all, why is this a function at all? we dont need to call this anywhere. We just need the simple values of things. This just has to be a javascript object with keys like maxloan, maxshares etc and their corresponding values. Then export that object.